packages/devtools_app/lib/src/shared/diagnostics/primitives/source_location.dart

```diff
@@ -69,7 +52,6 @@ class SourcePosition {
     );
   }
 
-  final String? file;
```

Comment:
it might be better to add this to toString

Reply:
Sure, do you have any preference regarding the format (ex. '$file:$line:$column' or '$file\n$line:$column')?

Reply:
'$file:$line:$column'  SGTM

Reply:
What bothers me is that file is not passed anywhere and is not used in `bool operator ==(other) {` and `int get hashCode =>`. On top of that any `toString` call will start with `null:`.

Are you sure this field should be restored?

Reply:
@elliette does this seem like a bug that we are not using the `file` parameter anywhere? I suspect this would cause weird equality issues where two `SourcePosition` objects appear identical because they have the same `line` & `column` & `tokenPos` but they are actually from two different files.

Reply:
I don't think this is a bug. It looks like `file` parameter was added in PR https://github.com/flutter/devtools/pull/3448 to combine two definitions of `SourcePosition` in two different files (`debugger_model` and `diagnostics_node`), the second of which had `file` as a parameter but never used it. 

`SourcePosition` is usually used in conjunction with some other class that contains file information (for example `ScriptRef` in [`StackFrameAndSourcePosition`](https://github.com/flutter/devtools/blob/6a2271a81ecea41aeb77ffed617587f1cceccc3d/packages/devtools_app/lib/src/screens/debugger/debugger_model.dart#L189)).

Reply:
Okay thanks for the input - @incendial why don't we just remove this parameter completely then as you originally did.